### ISSUE TITLE
chore: remove scripts from composer.json

### ISSUE THREAD

**Issue by mcdurdin** [keymanapp/keyman.com#493]
> Ah, I added the one in composer.json in #389 for feature parity with help.keyman.com

Hmm, so we should probably remove from all then. Let's keep composer.json minimal, remove all scripts, seeing as we do them in ci.yml now?

I see we never call `composer run test` either, which means we should probably add `vendor/bin/phpunit --testdox` to ci.yml and/or Dockerfile?

How do we run lint and phpunit tests locally?

_Originally posted by @mcdurdin in https://github.com/keymanapp/keyman.com/issues/492#issuecomment-2450982375_

1. Remove the scripts from composer.json
2. Make it possible to lint and phpunit test from local dev machine
3. Consolidate ci.yml and build.sh test calls (ci.yml should be able to call build.sh?)

This should also be done on help.keyman.com, elsewhere as well?

**Comment by darcywong00** [comment]
> How do we run lint and phpunit tests locally?

I thought one of the intents of putting everything into Docker is that the dev environment doesn't need to install anything else (Composer, PHP, phpunit, etc)

**Comment by mcdurdin** [comment]
> > How do we run lint and phpunit tests locally?
> 
> I thought one of the intents of putting everything into Docker is that the dev environment doesn't need to install anything else (Composer, PHP, phpunit, etc)

Yeah, but we still need to be able to trigger them, even if they run in the container, right? How do we do that?

**Comment by darcywong00** [comment]
> Yeah, but we still need to be able to trigger them, even if they run in the container, right? How do we do that?

Maybe a pattern like

https://github.com/keymanapp/shared-sites/blob/2a4ae1693880e168910c778c24a996e77993ca8c/_common/docker.inc.sh#L123-L133
